Adyen/Assets/ro-RO.lproj/Localizable.strings

```diff
@@ -65,6 +65,9 @@
 "adyen.voucher.readInstructions" = "Citiți instrucțiunile";
 "adyen.voucher.saveImage" = "Save as image";
 "adyen.voucher.finish" = "Finish";
+"adyen.pix.instructions" = "";
```

Comment:
I am wandering - if you do not provide this empty keys it will fallback to english automatically, right?

Reply:
removed those empty strings